include/scrimmage/entity/External.h

```diff
@@ -187,12 +188,18 @@ class External {
                 return;
             }
 
+            if (!step_called_at_least_once_) {
```

Comment:
Can we add an initial time to the init function so we don't run into this situation?